RecoTracker/PixelTrackFitting/python/PixelTracks_cff.py

```diff
@@ -103,31 +103,57 @@
+    maxNumberOfTuples   = str(32 * 1024),   # this couul be much lower (2.1k, these are quads)
+)
 phase2_tracker.toReplaceWith(pixelTracksAlpaka,_pixelTracksAlpakaPhase2.clone())
-(pp_on_AA & ~phase2_tracker).toReplaceWith(pixelTracksAlpaka, _pixelTracksAlpakaHIonPhase1.clone())
+phase2_tracker.toModify(pixelTracksAlpaka,
+    maxNumberOfDoublets = str(5*512*1024),
+    maxNumberOfTuples = str(256 * 1024),
+    avgHitsPerTrack = 7.0,
+    avgCellsPerHit = 6,
+    avgCellsPerCell = 0.151,
+    avgTracksPerCell = 0.040,
+    cellPtCut = 0.85,
+    cellZ0Cut = 7.5,
+    minYsizeB1 = 25,
+    minYsizeB2 = 15,
+    maxDYsize12 = 12,
+    maxDYsize = 10,
+    maxDYPred = 20,
+)
+
+
+(pp_on_AA & ~phase2_tracker).toModify(pixelTracksAlpaka,
+    maxNumberOfDoublets = str(6 * 512 *1024),   # this could be 2.3M
+    maxNumberOfTuples = str(256 * 1024),        # this could be 4.7
+    avgHitsPerTrack = 5.0,
+    avgCellsPerHit = 40,
+    avgCellsPerCell = 0.07,                     # with maxNumberOfDoublets ~= 3.14M; 0.02  for HLT HI on 2024 HI Data 
+    avgTracksPerCell = 0.03,                    # with maxNumberOfDoublets ~= 3.14M; 0.005 for HLT HI on 2024 HI Data
+    cellPtCut = 0.5,                            # setup currenlty used @ HLT (was 0.0) 
+    cellZ0Cut = 8.0,                            # setup currenlty used @ HLT (was 10.0) 
+)
 
 # pixel tracks SoA producer on the cpu, for validation
 pixelTracksAlpakaSerial = makeSerialClone(pixelTracksAlpaka,
     pixelRecHitSrc = 'siPixelRecHitsPreSplittingAlpakaSerial'
 )
 
 # legacy pixel tracks from SoA
-from  RecoTracker.PixelTrackFitting.pixelTrackProducerFromSoAAlpakaPhase1_cfi import pixelTrackProducerFromSoAAlpakaPhase1 as _pixelTrackProducerFromSoAAlpakaPhase1
-from  RecoTracker.PixelTrackFitting.pixelTrackProducerFromSoAAlpakaPhase2_cfi import pixelTrackProducerFromSoAAlpakaPhase2 as _pixelTrackProducerFromSoAAlpakaPhase2
-from  RecoTracker.PixelTrackFitting.pixelTrackProducerFromSoAAlpakaHIonPhase1_cfi import pixelTrackProducerFromSoAAlpakaHIonPhase1 as _pixelTrackProducerFromSoAAlpakaHIonPhase1
+from  RecoTracker.PixelTrackFitting.pixelTrackProducerFromSoAAlpaka_cfi import pixelTrackProducerFromSoAAlpaka as _pixelTrackProducerFromSoAAlpaka
 
-(alpaka & ~phase2_tracker).toReplaceWith(pixelTracks, _pixelTrackProducerFromSoAAlpakaPhase1.clone(
+(alpaka).toReplaceWith(pixelTracks, _pixelTrackProducerFromSoAAlpaka.clone(
     pixelRecHitLegacySrc = "siPixelRecHitsPreSplitting",
 ))
 
-(alpaka & phase2_tracker).toReplaceWith(pixelTracks, _pixelTrackProducerFromSoAAlpakaPhase2.clone(
-    pixelRecHitLegacySrc = "siPixelRecHitsPreSplitting",
-))
-
-(alpaka & ~phase2_tracker & pp_on_AA).toReplaceWith(pixelTracks, _pixelTrackProducerFromSoAAlpakaHIonPhase1.clone(
-    pixelRecHitLegacySrc = "siPixelRecHitsPreSplitting",
-))
+# (alpaka & stripNtupletFit).toModify(pixelTracks, 
+#         useStripHits = True)
```

Comment:
`stripNtupletFit` is not defined (in this file)... should these lines just be dropped ?